src/fibonacci.js

```diff
@@ -0,0 +1,34 @@
+export default function fibonacci( position ) {
+  if( typeof position !== 'number' ){
```

Comment:
This code works I tried it out. 

But to address your comments. both ways work but yeah your only going to do those checks once so you might as well implement it the way that you did. Good Job